colin/core/config/config.py

```diff
@@ -0,0 +1,116 @@
+import json
+import os
+
+from six import iteritems
+
+from colin.core.loader import load_check_implementation
+from colin.core.target import is_compatible
+
+CONFIG_DIRECTORY = "/usr/share/colin/"
```

Comment:
I would like to have a special file for such variables, like `constant.py`. But it is only an idea.

Reply:
Good point. I will make a `constant.py`.

Reply:
Done.